<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Correspondence</class>
 <widget class="QWidget" name="Correspondence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>304</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>244</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>CorrespondenceEstimation:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rbtn_ce">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_keypoints">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../src.qrc">
            <normaloff>:/icon/resource/icon/snap-bounding-box-center.svg</normaloff>:/icon/resource/icon/snap-bounding-box-center.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>CorrespondenceRejector:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbtn_cr">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbox_cr">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Distance</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MedianDistance</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OneToOne</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OrganizedBoundary</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Polygon</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SampleConsensus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SurfaceNormal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Trimmed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>VarTrimmed</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>75</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>5</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>MaximumDistance:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dspin_max_dis">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>99999.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>MedianFactor:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dspin_factor">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>99999.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_16"/>
        <widget class="QWidget" name="page_15">
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_22">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>BoundaryNaNs：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spin_val"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Cardinality:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spin_cardinality">
              <property name="maximum">
               <number>99999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>SimilarityThreshold:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dspin_sim_thres">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>99999.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Iterations:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spin_iterations">
              <property name="maximum">
               <number>99999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_4">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>InlierThreshold:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dspin_inlier_thres">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>99999.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>MaximumIterations:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spin_max_iterations">
              <property name="maximum">
               <number>99999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QCheckBox" name="check_refine">
              <property name="text">
               <string>RefineModel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_save">
              <property name="text">
               <string>SaveInliers</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_5">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Threshold:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dspin_threshold">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>99999.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_6">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>OverlapRatio:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dspin_ratio">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>99999.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>MinCorrespondences:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spin_min_corre">
              <property name="maximum">
               <number>99999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_7">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>MinRatio:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dspin_min_ratio">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>99999.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>MaxRatio:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dspin_max_ratio">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>99999.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="btn_setTarget">
          <property name="text">
           <string>SetTarget</string>
          </property>
          <property name="icon">
           <iconset resource="../../src.qrc">
            <normaloff>:/icon/resource/icon/target.svg</normaloff>:/icon/resource/icon/target.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_setSource">
          <property name="text">
           <string>SetSource</string>
          </property>
          <property name="icon">
           <iconset resource="../../src.qrc">
            <normaloff>:/icon/resource/icon/source.svg</normaloff>:/icon/resource/icon/source.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn_preview">
          <property name="text">
           <string>Preview</string>
          </property>
          <property name="icon">
           <iconset resource="../../src.qrc">
            <normaloff>:/icon/resource/icon/preview.svg</normaloff>:/icon/resource/icon/preview.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_apply">
          <property name="text">
           <string>Apply</string>
          </property>
          <property name="icon">
           <iconset resource="../../src.qrc">
            <normaloff>:/icon/resource/icon/apply1.svg</normaloff>:/icon/resource/icon/apply1.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_reset">
          <property name="text">
           <string>Reset</string>
          </property>
          <property name="icon">
           <iconset resource="../../src.qrc">
            <normaloff>:/icon/resource/icon/reset.svg</normaloff>:/icon/resource/icon/reset.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../src.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbox_cr</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
